groclient/lib.py

```diff
@@ -576,6 +576,39 @@ def get_data_points(access_token, api_host, **selection):
     return list_of_series_to_single_series(resp.json(), False, include_historical)
 
 
+def parse_params(allowed_selections, **selections):
+    params = {}
+    for key, value in list(selections.items()):
+        if key in allowed_selections:
+            params[groclient.utils.str_snake_to_camel(key)] = value
+    return params
+
+
+def get_data_points_v2_prime(access_token, api_host, allowed_selections, **selection):
```

Comment:
again, we should not be passing allowed_selections

Reply:
done